atox/src/main/kotlin/ActionReceiver.kt

```diff
@@ -30,8 +30,16 @@ import ltd.evilcorp.domain.tox.PublicKey
 import ltd.evilcorp.domain.tox.Tox
 
 const val KEY_TEXT_REPLY = "text_reply"
-const val KEY_CALL = "accept_or_reject_call"
 const val KEY_CONTACT_PK = "contact_pk"
+const val KEY_ACTION = "key_action"
+
+enum class Action {
+    CallAccept,
+    CallEnd,
+    CallIgnore,
+    CallReject,
+    MarkAsRead,
+}
```

Comment:
I made them alphabetical (prefixed by group if needed) because that's less to maintain, and it's easier to explain than more subjective orderings, so won't change this even if another order might feel nicer. :P

Reply:
OK